elements/rh-scheme-toggle/rh-scheme-toggle.ts

```diff
@@ -0,0 +1,94 @@
+import { html, isServer, LitElement } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+
+import '@rhds/elements/rh-icon/rh-icon.js';
+import { observes } from '@patternfly/pfe-core/decorators.js';
+
+import styles from './rh-scheme-toggle.css';
+// import visuallyHidden from './visually-hidden.css';
+
+declare global {
+  interface Storage {
+    rhdsColorScheme: 'light' | 'dark' | 'light dark';
+  }
+}
+
+type Scheme = 'light' | 'dark' | 'light dark';
+
+/**
+ * Scheme Toggle
+ */
+@customElement('rh-scheme-toggle')
+export class RhSchemeToggle extends LitElement {
+  static styles = [styles];
+
+  @property({ reflect: true }) scheme?: Scheme = globalThis.localStorage
+      ?.rhdsColorScheme as Scheme;
+
+  render() {
+    const isLight = this.scheme === 'light';
+    const isDark = this.scheme === 'dark';
+    const isSystem = (this.scheme === 'light dark') || (this.scheme === undefined);
```

Comment:
might be worth checking this against weird cases: 'light dark' vs 'dark light'
```suggestion
    const isSystem = (this.scheme.contains('light') && this.scheme.contains('dark')) || (this.scheme === undefined);
```